hackathon_site/dashboard/frontend/src/api/helpers.ts

```diff
@@ -69,6 +69,20 @@ export const teamOrderListSerialization = (
                     id: order.id,
                     status: order.status,
                     hardwareInTableRow,
+                    createdTime:
+                        "Created at: " +
+                        order.created_at.slice(0, 10) +
+                        " " +
+                        "at " +
+                        order.created_at.slice(11, 16) +
+                        "\xa0\xa0\xa0\xa0\xa0",
```

Comment:
What is the purpose of the \xao\xao\xao?

Reply:
Apologies for not leaving comments explaining what this is for. For visibility during testing, I added 
"\xa0" [a character that represents non-breaking space] to visually separate the createdAt and updatedAt times (since they are displayed on the same line). I've commented these out in the latest commit